modules/Layout/Header/SearchWidget/components/CategoriesList.tsx

```diff
@@ -44,23 +42,12 @@ function CategoriesList({ filteredCategories }: Props) {
             </ul>
 
             {filteredCategories.length > INITIAL_ITEMS_SHOWN && (
-                <Button
-                    onClick={toggleCategories}
-                    variation="navigation"
-                    className={classNames(styles.link, styles.viewMoreCategoriesLink)}
-                >
+                <Button onClick={toggleCategories} variation="navigation" className={styles.link}>
                     {showAllCategories ? (
                         <FormattedMessage {...translations.search.viewLess} />
                     ) : (
                         <FormattedMessage {...translations.search.viewMore} />
                     )}
-                    <IconSearchCaret
-                        width={12}
-                        height={12}
-                        className={classNames(styles.caret, {
-                            [styles.caretOpen]: showAllCategories,
```

Comment:
This class name should also be removed from the stylesheet

Reply:
Sure will remove this